config/zephyr/Kconfig

```diff
@@ -131,6 +170,20 @@ config APP_LINK_WITH_CHIP
 	  Add Connected Home over IP header files to the 'app' include path
 	  and link the 'app' with Connected Home over IP libraries.
 
+config CHIP_OTA_IMAGE_BUILD
+	bool "Generate OTA image"
+	help
+	  Enable building OTA (Over-the-air update) image. When this option is
+	  enabled, the resulting OTA image file will be created at
+	  build-dir/zephyr/matter-ota.bin.
```

Comment:
Correct me if I'm wrong, but isn't the matter-ota.bin name given by the nrfconnect script? So for other Zephyr platforms, like Telink it is not true.

Reply:
Right.

Reply:
Though... I'm not sure what path Telink will use. I wouldn't be surprised if it was the same. :)

Reply:
Maybe another kconfig specifying this path would help?

Reply:
I'll remove the comment. Not sure if there's much value in allowing to define this path. No other Zephyr paths are configurable I think.

Reply:
Done